airflow/www/views.py

```diff
@@ -332,6 +332,47 @@ def get_downstream(task):
     ]
 
 
+def dag_query_for_key(sorting_key):
+    """Maps sorting key param in the URL params to DB queries on appropriate attributes."""
+    dag_query_key_map = {
+        'dag_id': DagModel.dag_id,
+        'owner': DagModel.owners,
+        'schedule': DagModel.next_dagrun,
```

Comment:
I think I'd rather these values matched the column names -- we're already having to provide the "Readable String" and the order by in the template.

Reply:
Done